src/Core/Domain/Title/Exception/TitleConstraintException.php

```diff
@@ -32,4 +32,7 @@
 class TitleConstraintException extends TitleException
 {
     public const INVALID_ID = 1;
+
```

Comment:
This is wierd that phpcsfixer does not complain about the extra line here, or the missing line l37

Reply:
:heavy_check_mark: